Headers/DebugServer2/Architecture/X86_64/CPUState.h

```diff
@@ -238,13 +238,14 @@ struct CPUState64 {
     gp.r14 = regs[14];
     gp.r15 = regs[15];
     gp.rip = regs[16];
-    gp.eflags = regs[17];
-    gp.cs = regs[18];
-    gp.ss = regs[19];
-    gp.ds = regs[20];
-    gp.es = regs[21];
-    gp.fs = regs[22];
-    gp.gs = regs[23];
+    // Truncate the value to 32-bit as these are 32-bit registers.
```

Comment:
I guess I'm a bit confused about the point  of CPUState64 being a union. Why not simply
memcpy(gp.regs, regs, sizeof(gp.regs));
and avoid the need to remember which of these need to be truncated, etc?